src/math/numbers.c

```diff
@@ -177,3 +173,18 @@ uint32_t crc32(uint32_t base, const void *data, uint32_t bytes)
 
 	return ~crc;
 }
+
+/**
+ * \brief Calculate which power of 2 the alignment is.
+ * \param[in] alignment The component copy frame alignment.
+ */
+uint16_t alignment_shift_index(uint16_t alignment)
```

Comment:
In numbers.c it would be good if the function has a generic math function name. It looks like the function calculates floor(log2(alignment)), so would name like log2_uint16() make sense?